Header header

bool starting
float32 distance
uint8 type

uint8 TYPE_RED       = 0
uint8 TYPE_RED_AMBER = 1
uint8 TYPE_AMBER     = 2
uint8 TYPE_GREEN     = 3